src/org/usfirst/frc/team2335/robot/commands/AutoDriveTo.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoDriveTo extends Command
+{
+    public AutoDriveTo()
+    {
+        requires(Robot.driveTrain);
+        requires(Robot.ultraSound);
+    }
+
+    protected void initialize()
+    {
+    	
+    }
+
+    protected void execute()
+    {
+    	
+    }
+
+    // Make this return true when this Command no longer needs to run execute()
+    protected boolean isFinished()
+    {
+        return Robot.ultraSound.atZeroInches();
```

Comment:
Renaming to "frameAtZeroInches" and adding description to the function's documentation explaining that it's for when the frame of the robot is zero inches away from an obstacle would work best, I think. That way people will both know what's going on if they look at the function in detail, and if they just see it used somewhere.